src/main/home/send/SendOverview/index.tsx

```diff
@@ -1,52 +1,401 @@
+          <View
+            style={[
+              styles.overview,
+              {
+                borderColor: isDarkMode
+                  ? styledColors.darkGray
+                  : styledColors.border,
+                backgroundColor: isDarkMode
+                  ? styledColors.dark
+                  : styledColors.white,
+              },
+            ]}
+          >
+            <View style={styles.container}>
+              <Text
+                style={[
+                  styles.amountUnit,
+                  {
+                    color: isDarkMode ? styledColors.gray : styledColors.black,
+                  },
+                ]}
+              >
+                {eqInput.value} {eqInput.unit}
+              </Text>
+              <QuaiPayInputDisplay
+                value={input.value}
+                suffix={` ${input.unit}`}
+              />
+              <TouchableOpacity onPress={onSwap} style={[styles.exchangeUnit]}>
+                <Text
+                  style={{
+                    color: isDarkMode ? styledColors.white : styledColors.black,
+                  }}
+                >
+                  {input.unit}
+                </Text>
+                <ExchangeIcon
+                  color={isDarkMode ? styledColors.white : styledColors.black}
+                />
+              </TouchableOpacity>
+            </View>
+            <View
+              style={[
+                styles.border,
+                {
+                  backgroundColor: isDarkMode
+                    ? styledColors.darkGray
+                    : styledColors.border,
+                },
+              ]}
+            />
+            <Text style={styles.date}>{renderDate()}</Text>
+            <View style={styles.receiver}>
+              <QuaiPayText style={styles.username} type="paragraph">
+                {t('common:to')} {username}
+              </QuaiPayText>
+              <Text style={styles.wallet}>{abbreviateAddress(address)}</Text>
+            </View>
+            <View
+              style={[
+                styles.border,
+                {
+                  backgroundColor: isDarkMode
+                    ? styledColors.darkGray
+                    : styledColors.border,
+                },
+              ]}
+            />
+            <View style={styles.detailsContainer}>
+              <View style={styles.details}>
+                <View style={styles.detailLabel}>
+                  <QuaiPayText type="paragraph">
+                    {t('home.send.sending')}
+                  </QuaiPayText>
+                </View>
+                <View>
+                  <Text style={styles.unit}>
+                    {eqInput.value} {eqInput.unit}
+                  </Text>
+                  <Text style={styles.unitUSD}>
+                    {input.unit === Currency.USD
+                      ? `$${input.value} ${input.unit}`
+                      : `${input.value} ${input.unit}`}
```

Comment:
this validation could be moved just to add the `$` right?

```suggestion
                    {input.unit === Currency.USD && '$'}{
                      `${input.value} ${input.unit}`}
```